src/Chapter4.hs

```diff
@@ -629,7 +684,7 @@ Can you implement a monad version of AND, polymorphic over any monad?
 🕯 HINT: Use "(>>=)", "pure" and anonymous function
 -}
 andM :: (Monad m) => m Bool -> m Bool -> m Bool
-andM = error "andM: Not implemented!"
+andM m1 m2 = m1 >>= (\x -> if x == True then m2 else pure False )
```

Comment:
The `x` value is already of type `Bool`, so no need to compare it with `True` :slightly_smiling_face: 
```suggestion
andM m1 m2 = m1 >>= (\x -> if x then m2 else pure False )
```

Reply:
thanks, sometimes it happens involuntarily :)